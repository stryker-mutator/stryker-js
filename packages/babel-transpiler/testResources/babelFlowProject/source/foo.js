function foo(one: any, two: number, three?): string {}