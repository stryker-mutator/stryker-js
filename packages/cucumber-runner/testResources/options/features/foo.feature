Feature: Foo

  @foo
  Scenario: Foo should bar
    Then foo should "bar"

  @foo
  Scenario Outline: Foo should bar outline
    Then foo should "<name>"

    Examples:
      | name |
      | bar  |
      | baz  |
