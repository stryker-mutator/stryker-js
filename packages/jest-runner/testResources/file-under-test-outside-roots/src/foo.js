// Manually instrumented out of lazy-ness ðŸ˜…
if (process.env.__STRYKER_ACTIVE_MUTANT__ === '1'){
  module.exports = 0;
} else {
  module.exports = 42;
}
