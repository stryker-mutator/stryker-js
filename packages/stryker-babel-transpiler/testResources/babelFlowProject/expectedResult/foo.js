function foo(one, two, three) {}