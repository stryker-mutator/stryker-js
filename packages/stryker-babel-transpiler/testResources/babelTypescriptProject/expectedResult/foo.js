export const foo = 4;