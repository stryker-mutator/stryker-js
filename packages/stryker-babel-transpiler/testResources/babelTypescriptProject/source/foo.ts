export const foo: number = 4;