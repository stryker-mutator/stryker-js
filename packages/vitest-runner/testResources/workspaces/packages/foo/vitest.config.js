import { defineProject } from 'vitest/config';

export default defineProject({
  test: {
    setupFiles: ['vitest.setup.js'],
  },
});
