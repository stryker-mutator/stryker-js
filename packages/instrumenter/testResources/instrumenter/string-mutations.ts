require('foo'); // no mutation
import 'foo'; // no mutation

class Foo {
  public 'bar-baz': string; // no mutation
}

