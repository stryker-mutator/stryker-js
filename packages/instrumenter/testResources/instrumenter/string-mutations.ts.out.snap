// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`instrumenter integration should be able to instrument string literals in different places 1`] = `
"require('foo'); // no mutation


import 'foo'; // no mutation

class Foo {
  public 'bar-baz': string; // no mutation

}"
`;
